File: python3_solution_set0/283. Move Zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        
        #count  = 0
        ans = []
        for x in range(len(nums)):
            if nums[x]!= 0:
                ans.append(nums[x])
                
        n = len(nums)
        m = len(ans)
        
        for x in range(n-m):
            ans.append(0)
            
        for x in range(len(ans)):
            nums[x] = ans[x]
                
        
        

File: go_solution_set1/283. Move Zeroes.go
func moveZeroes(nums []int)  {
    ans := []int{}
    cnt := 0
    for _, val := range nums{
        if val == 0{
            cnt+=1
        } else{
            ans = append(ans, val)
        }
    }

    for i:=0; i<cnt;i++{
        ans = append(ans,0)
    }

    for i:=0; i<len(nums);i++{
        nums[i] = ans[i]
    }
    
}
